codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/localservice/DafnyPythonLocalServiceProtocolGenerator.java

```diff
@@ -496,7 +496,7 @@ private void generateErrorResponseDeserializerSection(
             writer.write(
               """
               else:
-                  return OpaqueError(obj=error)"""
+                  return OpaqueError(obj=error, alt__text=repr(error))"""
```

Comment:
```suggestion
                  return OpaqueError(obj=error, alt_text=repr(error))"""
```

Reply:
This change still needs to go in;
`error` is a Dafny error with `alt__text`,
but `OpaqueError` is the native type with `alt_text`